ts-tests/tests/test-revert-receipt.ts

```diff
@@ -0,0 +1,118 @@
+
+	it("should provide a tx receipt after successful deployment", async function () {
+		this.timeout(15000);
+		const GOOD_TX_HASH = '0xae813c533aac0719fbca4db6e3bb05cfb5859bdeaaa7dc5c9dbd24083301be8d';
+
+		const tx = await context.web3.eth.accounts.signTransaction(
+			{
+				from: GENESIS_ACCOUNT,
+				data: GOOD_BYTECODE,
+				value: "0x00",
+				gasPrice: "0x01",
+				gas: "0x100000",
+			},
+			GENESIS_ACCOUNT_PRIVATE_KEY
+		);
+
+		expect(
+			await customRequest(context.web3, "eth_sendRawTransaction", [tx.rawTransaction])
+		).to.deep.equal({
+			id: 1,
+			jsonrpc: "2.0",
+			result: GOOD_TX_HASH,
+		});
+
+		// Verify the receipt exists after the block is created
+		//TODO Actually, why doesn't this receipt have a status in it?
+		// I guess because the RPC handler in eth.rs sets `status_code: None`??
+		await createAndFinalizeBlock(context.web3);
+		expect(
+			await customRequest(context.web3, "eth_getTransactionReceipt", [GOOD_TX_HASH])
+		).to.deep.equal({
+			id: 1,
+			jsonrpc: "2.0",
+			result: {
+				"blockHash": "0xf543706adc989641d066cf0831374a8b33aad1dd91dc1f18ee75786d50d478e2",
+				"blockNumber": "0x1",
+				"contractAddress": "0xc2bf5f29a4384b1ab0c063e1c666f02121b6084a",
+				"cumulativeGasUsed": "0x1069f",
+				"from": "0x6be02d1d3665660d22ff9624b7be0551ee1ac91b",
+				"gasUsed": "0x1069f",
+				"logs": [],
+				"logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+				"root": "0x0000000000000000000000000000000000000000000000000000000000000000",
+				"to": null,
+				"transactionHash": GOOD_TX_HASH,
+				"transactionIndex": "0x0",
+			}
+		});
+	});
+
+	it("should provide a tx receipt after failed deployment", async function () {
+		this.timeout(15000);
+		// Transaction hash depends on which nonce we're using
+		//const FAIL_TX_HASH = '0x89a956c4631822f407b3af11f9251796c276655860c892919f848699ed570a8d'; //nonce 1
+		const FAIL_TX_HASH = '0x640df9deb183d565addc45bdc8f95b30c7c03ce7e69df49456be9929352e4347'; //nonce 2
+
+		const tx = await context.web3.eth.accounts.signTransaction(
+			{
+				from: GENESIS_ACCOUNT,
+				data: FAIL_BYTECODE,
+				value: "0x00",
+				gasPrice: "0x01",
+				gas: "0x100000",
+			},
+			GENESIS_ACCOUNT_PRIVATE_KEY
+		);
+
+		expect(
+			await customRequest(context.web3, "eth_sendRawTransaction", [tx.rawTransaction])
```

Comment:
Any reason not to use `context.web3.eth.sendRawTransaction` ?

Reply:
I followed the style of the existing tests.